### PR TITLE
Tiny tweaks

### PR DESCRIPTION
**Pull request by ekie**

### CODE REVIEW

File: README.asciidoc [hunk 1]
```diff
@@ -99,8 +100,8 @@ Building
 
 Kakoune dependencies are:
 
- * A C\+\+11 compliant compiler (GCC >= 5 or clang >= 3.4) along with its
-   associated C\+\+ standard library (libstdc\+\+ or libc\+\+)
+ * A {cpp} compliant compiler (GCC >= 5 or clang >= 3.4) along with its
```
Comment (Delapouite) [inline README.asciidoc]: Why removing the explicit version number in your adjustment?

File: README.asciidoc [hunk 2]
```diff
@@ -99,8 +100,8 @@ Building
 
 Kakoune dependencies are:
 
- * A C\+\+11 compliant compiler (GCC >= 5 or clang >= 3.4) along with its
-   associated C\+\+ standard library (libstdc\+\+ or libc\+\+)
+ * A {cpp}11 compliant compiler (GCC >= 5 or clang >= 3.4) along with its
```
Comment (mawww) [inline README.asciidoc]: The readme is expected to be readable as text as well, so I'd rather keep `C++` written as-is in the source.

Comment (ekie) [inline README.asciidoc]: I understand but i think the way it gets rendered on github is weird:

`A C\\+11 compliant compiler (GCC >= 5 or clang >= 3.4) along with its associated C\\+ standard library (libstdc\\ or libc\\)`

Comment (mawww) [inline README.asciidoc]: OK yeah, its pretty tricky to fix due to how asciidoc works, fine...

**Pull request merged**